File: script/make/protocol/json/ItemProtocol.js
const itemProtocol = {
    read: {
        11101: [
            {name: "list", type: "list", comment: "道具列表", explain: [
                {name: "itemNo", type: "u64", comment: "物品编号", explain: []},
                {name: "itemId", type: "u32", comment: "物品ID", explain: []},
                {name: "type", type: "u8", comment: "类型", explain: []},
                {name: "number", type: "u16", comment: "数量", explain: []}
            ]}
        ],
        11102: [
            {name: "list", type: "list", comment: "背包列表", explain: [
                {name: "itemNo", type: "u64", comment: "物品编号", explain: []},
                {name: "itemId", type: "u32", comment: "物品ID", explain: []},
                {name: "type", type: "u8", comment: "类型", explain: []},
                {name: "number", type: "u16", comment: "数量", explain: []}
            ]}
        ],
        11103: [
            {name: "list", type: "list", comment: "仓库列表", explain: [
                {name: "itemNo", type: "u64", comment: "物品编号", explain: []},
                {name: "itemId", type: "u32", comment: "物品ID", explain: []},
                {name: "type", type: "u8", comment: "类型", explain: []},
                {name: "number", type: "u16", comment: "数量", explain: []}
            ]}
        ],
        11104: [
            {name: "list", type: "list", comment: "删除列表", explain: [
                {name: "itemNo", type: "u64", comment: "物品编号", explain: []},
                {name: "type", type: "u8", comment: "类型", explain: []}
            ]}
        ]
    },
    write: {
        11101: [],
        11102: [],
        11103: []
    }
};

File: script/make/protocol/cs/meta/ItemProtocol.cs
using List = System.Collections.Generic.List<System.Object>;
using Map = System.Collections.Generic.Dictionary<System.String, System.Object>;

public static class ItemProtocol
{
    public static Map GetMeta()
    {
        return new Map()
        {
            {"11101", new Map() {
                {"comment", "道具列表"},
                {"write", new List() {
                    new Map() { {"name", "data"}, {"type", "tuple"}, {"comment": ""}, {"explain": new List() {

                    }}}
                }},
                {"read", new List() {
                    new Map() { {"name", "data"}, {"type", "list"}, {"comment", ""}, {"explain", new List() {
                        new Map() { {"name", "item"}, {"type", "record"}, {"comment": ""}, {"explain": new List() {
                            new Map() { {"name", "itemNo"}, {"type", "u64"}, {"comment", "物品编号"}, {"explain", new List()} },
                            new Map() { {"name", "itemId"}, {"type", "u64"}, {"comment", "物品ID"}, {"explain", new List()} },
                            new Map() { {"name", "type"}, {"type", "u8"}, {"comment", "类型"}, {"explain", new List()} },
                            new Map() { {"name", "number"}, {"type", "u16"}, {"comment", "数量"}, {"explain", new List()} }
                        }}}
                    }}}
                }}
            }},
            {"11102", new Map() {
                {"comment", "背包列表"},
                {"write", new List() {
                    new Map() { {"name", "data"}, {"type", "tuple"}, {"comment": ""}, {"explain": new List() {

                    }}}
                }},
                {"read", new List() {
                    new Map() { {"name", "data"}, {"type", "list"}, {"comment", ""}, {"explain", new List() {
                        new Map() { {"name", "item"}, {"type", "record"}, {"comment": ""}, {"explain": new List() {
                            new Map() { {"name", "itemNo"}, {"type", "u64"}, {"comment", "物品编号"}, {"explain", new List()} },
                            new Map() { {"name", "itemId"}, {"type", "u64"}, {"comment", "物品ID"}, {"explain", new List()} },
                            new Map() { {"name", "type"}, {"type", "u8"}, {"comment", "类型"}, {"explain", new List()} },
                            new Map() { {"name", "number"}, {"type", "u16"}, {"comment", "数量"}, {"explain", new List()} }
                        }}}
                    }}}
                }}
            }},
            {"11103", new Map() {
                {"comment", "仓库列表"},
                {"write", new List() {
                    new Map() { {"name", "data"}, {"type", "tuple"}, {"comment": ""}, {"explain": new List() {

                    }}}
                }},
                {"read", new List() {
                    new Map() { {"name", "data"}, {"type", "list"}, {"comment", ""}, {"explain", new List() {
                        new Map() { {"name", "item"}, {"type", "record"}, {"comment": ""}, {"explain": new List() {
                            new Map() { {"name", "itemNo"}, {"type", "u64"}, {"comment", "物品编号"}, {"explain", new List()} },
                            new Map() { {"name", "itemId"}, {"type", "u64"}, {"comment", "物品ID"}, {"explain", new List()} },
                            new Map() { {"name", "type"}, {"type", "u8"}, {"comment", "类型"}, {"explain", new List()} },
                            new Map() { {"name", "number"}, {"type", "u16"}, {"comment", "数量"}, {"explain", new List()} }
                        }}}
                    }}}
                }}
            }},
            {"11106", new Map() {
                {"comment", "使用物品"},
                {"write", new List() {
                    new Map() { {"name", "data"}, {"type", "tuple"}, {"comment": ""}, {"explain": new List() {
                        new Map() { {"name", "itemNo"}, {"type", "u64"}, {"comment", "物品编号"}, {"explain", new List()} },
                        new Map() { {"name", "number"}, {"type", "u16"}, {"comment", "数量"}, {"explain", new List()} },
                        new Map() { {"name", "type"}, {"type", "u8"}, {"comment", "类型"}, {"explain", new List()} }
                    }}}
                }},
                {"read", new List() {
                    new Map() { {"name", "data"}, {"type", "rst"}, {"comment", "结果"}, {"explain", new List()} }
                }}
            }}
        };
    }
}